guides/office/dress_code.md

```diff
@@ -0,0 +1,10 @@
+# Dress Code
+
+Made Tech is typically a casually dressed environment, however, we should keep in mind that we remain a professional organisation.
+
+This should be considered a rough guide as to acceptable dress in the Made Tech office. If you're working on-site or visiting a customer organisation you should be sensitive to their dress code and modify your dress accordingly. If in doubt please seek advice from another member of the team.
+
+## Generally not OK for the office:
+
+- Tracksuit bottoms / sweatpants
```

Comment:
I personally don't agree with this, when you're sitting down for 6+ hours its not always comfortable, hence why somedays I wear black tracksuit bottoms. I fully understand that if I see a client or visit a clients offices I wear jeans, I did this when I went to LAA and I dressed appropriately, however in the office I would like to feel as comfortable as possible while I sit and code for hours. @chrisblackburn

Reply:
Unfortunately we have customers visit unexpectedly or have to head to a meeting without much notice so we do need to create guidance based on that requirement.

Reply:
@chrisblackburn Could probably add a bit more of an explanation as to why we need to have a dress code?

Reply:
Added to the first paragraph on the need to project a level of professionalism at all times